codec-http2/src/main/java/io/netty/handler/codec/http2/CompressorHttp2ConnectionEncoder.java

```diff
@@ -55,6 +45,17 @@ public void onStreamRemoved(Http2Stream stream) {
     private final int compressionLevel;
     private final int windowBits;
     private final int memLevel;
+    private final Http2Connection.PropertyKey<CompressorHttp2ConnectionEncoder, EmbeddedChannel> propertyKey;
+
+    private final Http2ConnectionAdapter cleanUpListener = new Http2ConnectionAdapter() {
```

Comment:
Maybe we should delay the initialization of this to after `propertyKey` is initialized because it depends on it?